terms-of-service.md

```diff
@@ -1,3 +1,90 @@
-# Terms
+# Terms of Service
+
+---
+title: Terms
+order: 4
+---
+
+Hey there! We're Bloggify and we're happy that you are here, on our website. When we say **we**, **us** or **Bloggify** it's because that's who we are. We own and run the Bloggify website.
+
+By navigating on this website you agree to follow the the terms explained on this page. Please read and understand them.
+If you don't accept them, then we'll be sorry, but you will need to leave.
+
+You promise that information you give us is true, accurate and complete.
```

Comment:
@lex124 I guess it would be better to move these paragraphs (this current one and the next 2 ones) in the buying process section.